_posts/2020-03-04-eda-architecture2.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "An event driven architecture — part 2"
+excerpt: "Core design principles."
+date:   2020-03-04 12:53:00 +0100
+categories: architecture
+tags:  services event bus architecture
+---
+
+In the previous post in this series, we explained why we ditched our old architecture based on synchronous REST services for a completely asynchronous event driven architecture.
```

Comment:
Will it be easy to add a link to ease navigation from article to article?

I would also suggest, at the end of each article, listing every articles and update the text to a link once a given article has been published.